test/start-end-test.js

```diff
@@ -0,0 +1,313 @@
+'use strict';
+
+var sinon = require('sinon');
+var main = require('../package.json').main;
+var schedule = require('../' + main);
+
+var clock;
+
+module.exports = {
+	setUp: function (cb) {
+    clock = sinon.useFakeTimers();
+    cb();
+  },
+  'RecurrenceRule': {
+    'no endTime , startTime less than now': function (test) {
+       test.expect(3);
+
+      var job = new schedule.Job(function() {
```

Comment:
`test.expect(3);` should be aligned with the rest of the test code. The same for every test (I think)